idp_user/migrations/0002_auto_20220120_1427.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.14 on 2022-01-20 14:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('idp_user', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='userrole',
+            name='role',
+            field=models.CharField(choices=[('MercuryUser', 'MercuryUser')], max_length=125),
+        ),
+    ]
```

Comment:
Remove this file as this is a migration that applies only when the library is used inside Mercury, but is not a migration of the library itself.

Reply:
Done